test/systemtests/init_test.go

```diff
@@ -3,23 +3,25 @@ package systemtests
 import (
 	"flag"
 	"fmt"
+	"os"
+	. "testing"
+
 	"github.com/Sirupsen/logrus"
 	"github.com/contiv/contivmodel/client"
 	"github.com/contiv/remotessh"
 	. "gopkg.in/check.v1"
-	"os"
-	. "testing"
 )
 
 type systemtestSuite struct {
-	vagrant   remotessh.Vagrant
-	baremetal remotessh.Baremetal
-	cli       *client.ContivClient
-	nodes     []*node
-	fwdMode   string
-	basicInfo BasicInfo
-	hostInfo  HostInfo
-	globInfo  GlobInfo
+	vagrant       remotessh.Vagrant
+	baremetal     remotessh.Baremetal
+	cli           *client.ContivClient
+	nodes         []*node
+	fwdMode       string
+	netmasterPort string
```

Comment:
Should we put this in GlobalInfo Struct ?  This structure is for over all systemtest. Same comment for fwd mode field as well.